### PR TITLE
IDE-522 TextEditor - Background color to denote the target

### PR DESCRIPTION
**Pull request by dehilsterlexis**
Fixed IDE-522

Signed-off-by: David de Hilster <david.dehilster@lexisnexis.com>

### CODE REVIEW

**Comment by dehilsterlexis** [comment]
@GordonSmith for your review

File: docs/LanguageColor.xml [hunk 1]
```diff
@@ -408,16 +408,64 @@
...
+      <first>42</first>
+      <second>
+        <categoryid>42</categoryid>
+        <name>RoxieBackground</name>
+        <fore></fore>
+        <back>16771043</back>
+        <font></font>
+        <size></size>
+        <bold></bold>
+      </second>
```
Comment (GordonSmith) [inline docs/LanguageColor.xml]: These new colours should share a single category ID.

**Comment by GordonSmith** [comment]
Code looks ok - one question about the category ID.

File: docs/LanguageColor.xml (hunk 1 above)
Comment (dehilsterlexis) [inline docs/LanguageColor.xml]: Corrected and uploaded.

**Comment by dehilsterlexis** [comment]
@GordonSmith @richardkchapman ID fixed.

**Comment by GordonSmith** [comment]
Looks fine.

**Pull request merged**